### DISCUSSION TITLE
Your npm script commands are not working. even normal 'npx playwright test "particularfile"' command also not working.

### DISCUSSION THREAD

**Discussion by shirshGit** [akshayp7/playwright-typescript-playwright-test#24] [Q&A] [answered]
Commented this part of code
// if (!ENV || ![`qa`,`stage`, `dev`, `qaApi`, `devApi`].includes(ENV)) {
//   console.log(`Please provide a correct environment value like "npx cross-env ENV=qa|dev|qaApi|devApi"`);
//   process.exit();
// }

to see if I can simply execute tests groupwise(like @Smoke). its okay if test fails. But its showing 'no tests found.'

can you please let me know if I am doing anything wrong.
npx playwright test .\tests\Test1.test.ts --- this command also not working..its saying no test found

**Comment by akshayp7** [comment, marked as answer]
In playwright.config.ts, the part where "baseURL: testConfig[process.env.ENV]", provide your site url and try and let me know if it works
